share/spack/qa/run-flake8

```diff
@@ -21,9 +21,12 @@ fi
 changed=$(git diff --name-only develop... | grep '.py$')
 
 # Exempt url lines in changed packages from overlong line errors.
+# Exempt functions defined with '@when' decorator from redefinition errors.
```

Comment:
@adamjstewart: I think that this is a good idea as well.  I'll change the code during my next update.